cypress/components/help/help.cy.tsx

```diff
@@ -78,13 +63,16 @@ context("Tests for Help component", () => {
         "Warning: The tag <%s> is unrecognized in this browser. If you meant to render a React component, start its name with an uppercase letter.%s",
         asElem
       );
-      let errors;
+      let errors: any;
       beforeEach(() => {
         cy.window().then((win) => {
           errors = cy.spy(win.console, "error").as("spyWinConsoleError");
         });
       });
+
       it("should throw a console error when invalid html elements are used", () => {
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore:next-line
```

Comment:
For context for other reviewers, I suggested @divyajindel to instruct the TypeScript compiler to ignore this line.

TypeScript consumers will see a compile-time error if they attempt to pass an illegal value as normal, but for non-TypeScript consumers who don't have this ability, we need to make sure a runtime error is raised for this case as well, which is want the test is verifying.

Reply:
still not resolved